ethereumetl/cli/stream.py

```diff
@@ -50,8 +51,10 @@
 @click.option('-w', '--max-workers', default=5, show_default=True, type=int, help='The number of workers')
 @click.option('--log-file', default=None, show_default=True, type=str, help='Log file')
 @click.option('--pid-file', default=None, show_default=True, type=str, help='pid file')
+@click.option('--connection-url', default=None, show_default=True, type=str, help='Connection url for file, required for kafka')
```

Comment:
The `--output` field is usually where the connection url would go. To make it more explicit and at the same time consistent and backward compatible, I'd add another param --output-type where users could specify one of `ItemExporterType`. If `--output-type` is not specified, we try to determine it from value of `--output` as it's done in `determine_item_exporter_type` currently. So for existing output types users only need to specify `--output` and for kafka they'd specify `--output-type kafka --output <bootstrap_servers>

Reply:
Scratch that. It won't work since --output is a comma separated list of outputs which can be of different types

Reply:
How about we just embed `kafka` in the output e.g. `kafka:<connection_url>`?

Reply:
@medvedev1088 I liked the idea of having it single param, made the changes please check